docs/tutorials/google/start.ipynb

```diff
@@ -163,14 +163,24 @@
     }
    ],
    "source": [
+    "import os\n",
+    "\n",
     "# The Google Cloud Project id to use.\n",
     "project_id = \"\" #@param {type:\"string\"}\n",
-    "processor_id = \"pacific\" #@param {type:\"string\"}\n",
+    "processor_id = \"\" #@param {type:\"string\"}\n",
+    "\n",
+    "os.environ['GOOGLE_CLOUD_PROJECT'] = project_id\n",
```

Comment:
Should probably not set this until after calling `get_qcs_objects_for_notebook` below, since the project id could be unspecified here and resolved by `google.auth.default()` to something from gcloud config.

Reply:
Makes sense to move it below, but it looks like it's not getting resolved in Colab today and users aren't able to run the notebook, so we should make sure it gets defined.